scran-compute-sum-factors.R

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env Rscript 
+
+#computeSumFactors performs a scaling normalization of single-cell RNA-seq data by deconvolving size factors from cell pools. 
+#Note: cells should have non-zero library sizes.  
+#The assumption is that most genes are not differentially expressed (DE) between cells, 
+#such that any differences in expression across the majority of genes represents some technical bias that should be removed.
+
+suppressPackageStartupMessages(require(optparse))
+suppressPackageStartupMessages(require(workflowscriptscommon))
+
+# argument parsing 
+option_list = list(
+  make_option(
+    c("-i", "--input-sce-object"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = 'Path to the input SCE object in rds format'
+  ),
+    make_option(
+    c("-a", "--assay-type"),
+    action = "store",
+    default = "counts",
+    type = 'character',
+    help = 'Specify which assay values to use. Default: "logocunts'
```

Comment:
@rubenchazarra, 'logocunts' is a nice one, but you'll have to replace it, I'm afraid ;)